akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/ShardedDaemonProcessSettings.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding.typed
+
+import java.time.Duration
+
+import akka.actor.typed.ActorSystem
+import akka.annotation.ApiMayChange
+import akka.annotation.InternalApi
+import akka.util.JavaDurationConverters._
+import com.typesafe.config.Config
+
+import scala.concurrent.duration.FiniteDuration
+
+object ShardedDaemonProcessSettings {
```

Comment:
I think we should ApiMayChange this and other public bits for now?

Reply:
Yes, lets